packages/widget/src/controllers/wallet-manager/token-balance.ts

```diff
@@ -100,4 +119,25 @@ export class TokenBalanceController implements ReactiveController {
       this.host.requestUpdate();
     }.bind(this)();
   };
+
+  suscribeSubstrateBalanceUpdate = (resource: SubstrateResource): void => {
+    const { substrateProvider: apiPromise, accounts } = this.walletContext.value
+      ?.substrateWallet as SubstrateWallet;
+
+    const accountAddress = accounts![0].address;
+
+    void async function (this: TokenBalanceController) {
+      const balance = await getAssetBalance(
```

Comment:
try/catch so you can update loadingBalance?

Reply:
@mpetrunic I think we should also try/catch the evm ones, wdyt?